packages/lib-content/src/components/Article/Article.js

```diff
@@ -51,7 +51,7 @@ export default function Article({
         <Heading
           color={{ light: 'neutral-1', dark: 'accent-1' }}
           level={4}
-          size='1.5rem'
+          size={size !== 'small' ? '1.5rem' : '1.2rem'}
```

Comment:
This change also affects the Community section on the signed-in and signed-out homepages, but overall improves the Article component's responsiveness.

Reply:
Smart choice here. I think it helps with the visual hierarchy once you're at that screen size.